pkg/cloud/azure/services/internalloadbalancers/internalloadbalancers.go

```diff
@@ -157,13 +157,13 @@ func (s *Service) Delete(ctx context.Context, spec azure.Spec) error {
 		return errors.New("invalid internal load balancer specification")
 	}
 	klog.V(2).Infof("deleting internal load balancer %s", internalLBSpec.Name)
-	f, err := s.Client.Delete(ctx, s.Scope.ClusterConfig.ResourceGroup, internalLBSpec.Name)
+	f, err := s.Client.Delete(ctx, s.Scope.ResourceGroup().Name, internalLBSpec.Name)
```

Comment:
why do we use `f` here? Is it worth it to change to more descriptive variable names as we go on?

Reply:
I also know we use `f` elsewhere in this PR and the code, this is just where I noticed it

Reply:
Thanks for noticing this, @soggiest. I've updated the variables to `future`, which indicate that they are placeholders for future responses from Azure.